client/fs_endpoint_test.go

```diff
@@ -1897,6 +1899,15 @@ func TestFS_logsImpl_NoFollow(t *testing.T) {
 	case <-time.After(10 * time.Duration(testutil.TestMultiplier()) * streamBatchWindow):
 		t.Fatalf("did not receive data: got %q", string(received))
 	}
+
+	select {
+	case err := <-errCh:
+		if err != nil {
+			t.Fatalf("logsImpl: %v", err)
```

Comment:
FYI: we're trying to converge on `require.NoError(t, err)` for these sorts of checks, but this fine.